### DISCUSSION TITLE
clsx version ?

### DISCUSSION THREAD

**Discussion by sloonz** [mantinedev/mantine#6226] [Q&A] [answered]
Any good reason why clsx is declared as an exact version ("2.1.0") in mantine-core package.json, instead of "^2.1.0", or is it just a typo ?

It is a bit annoying to have both 2.1.0 installed from mantine and 2.1.1 installed from other deps.

**Comment by rtivital** [comment, marked as answer]
The version of clsx will be updated in one of the next releases.
